include/imu_vn_100/imu_vn_100.h

```diff
@@ -123,6 +124,9 @@ class ImuVn100 final : public rclcpp::Node {
   int hsi_output_;
   int hsi_converge_rate_;
 
+  bool ref_use_models_;
+  int ref_recalc_threshold_;
```

Comment:
This should be a uint32_t (so you'll probably have to `static_cast` during the `declare_parameter` in the constructor).  Also, the data sheet doesn't list a unit for this value, but the discussion heavily suggests that this is in meters; if that is the case, I'd prefer to have the units encoded into the parameter name and variable name (so `ref_recalc_threshold_m_`).

Reply:
All fair points. Fixed in 321aff1f5c3eff46c4dc3a98e1dbab586690963e.